### ISSUE TITLE
Swiper images flicker or don't appear at all on device. Works fine in simulator. Limited to RN > 0.54.x

### ISSUE THREAD

**Issue by jmknoll** [leecade/react-native-swiper#768]
### Which OS ?
iOS 11.3

### Version
Which versions are you using:

- react-native-swiper v1.5.13
- react-native v0.55.3

### Expected behaviour

Swiper should appear and not cause the app to crash.

### Actual behaviour

Swiper does not appear and causes the app to crash randomly.

### How to reproduce it>
To help us, please fork this component, modify one example in examples folder to reproduce your issue and include link here.

The swiper component runs normally in the simulator and on device when running from xcode, but when I build the app and distribute through iTunes connect, the swiper images flicker or do not appear at all, and the presence of the swiper causes the app to crash randomly.

This is new to RN 0.55.x. This bug did not appear in RN < 0.54.x

### Steps to reproduce
1. Archive app
2. Distribute through testflight
3. Open app and look at page with react-native-swiper

**Comment by havinhthai** [comment]
Same problem on IOS

**Comment by koswarabilly** [comment]
i am having the same issue on android which cause crash on real device after installing signed apk

**Comment by PerspectivesLab** [comment]
maybe one solution:

```
class SwiperTest extends Swiper { 
 componentWillReceiveProps (nextProps) {
    if (!nextProps.autoplay && this.autoplayTimer) clearTimeout(this.autoplayTimer)
// comment this line
    //this.setState(this.initState(nextProps, this.props.index !== nextProps.index))
}
}
```
now use SwiperTest  in your render function, it helped stoping bugs on unfocused component

**Comment by egunsoma** [comment]
Yes me too having this problem on Android, Swiper doesn't show up at all. Works on Android emulator.
@koswarabilly have you managed to solve this issue?

**Comment by DavitVosk** [comment]
Sma problem

**Comment by jmknoll** [comment]
For anyone still running into this problem, I would recommend switching over to [react-native-snap-carousel](https://github.com/archriss/react-native-snap-carousel). I've switched over my projects, and this library has worked more reliably for me. React-native-swiper doesn't appear to be maintained anymore.

**Comment by dhirajanand014** [comment]
> For anyone still running into this problem, I would recommend switching over to [react-native-snap-carousel](https://github.com/archriss/react-native-snap-carousel). I've switched over my projects, and this library has worked more reliably for me. React-native-swiper doesn't appear to be maintained anymore.

Even it has flickering issue when you enable the loop.
